tests/pipelines/main.nf.test

```diff
@@ -1102,4 +1104,109 @@ nextflow_pipeline {
             assert iridanext_metadata.isEmpty()
         }
     }
+
+    test("Test zipped output"){
```

Comment:
Thanks so much for checking the file contents inside the zip archive in this test. I really appreciate it 😄